cluster-up/cluster/kind-k8s-1.19/provider.sh

```diff
@@ -4,12 +4,23 @@ set -e
 
 DOCKER="${CONTAINER_RUNTIME:-docker}"
 
-export CLUSTER_NAME="kind-1.17"
+DEFAULT_CLUSTER_NAME="kind-1.19"
+DEFAULT_HOST_PORT=5000
+ALTERNATE_HOST_PORT=5001
+export CLUSTER_NAME=${CLUSTER_NAME:-$DEFAULT_CLUSTER_NAME}
+
+if [ $CLUSTER_NAME == $DEFAULT_CLUSTER_NAME ]; then
+    export HOST_PORT=$DEFAULT_HOST_PORT
+else
+    export HOST_PORT=$ALTERNATE_HOST_PORT
+fi
+
+TESTS_NS="${TESTS_NS:-kubevirt-test-default1}"
 
 function set_kind_params() {
-    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.17.2}"
-    export KIND_VERSION="${KIND_VERSION:-0.7.0}"
-    export KUBECTL_PATH="${KUBECTL_PATH:-/kind/bin/kubectl}"
+    export KIND_VERSION="${KIND_VERSION:-0.11.1}"
+    export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.19.11}"
```

Comment:
Please make sure to use the correct digest image take according to kind version
https://github.com/kubernetes-sigs/kind/releases/tag/v0.11.1

Reply:
Do you mean I should append the sha256 of node image, like
```
export KIND_NODE_IMAGE="${KIND_NODE_IMAGE:-kindest/node:v1.19.11@sha256:07db187ae84b4b7de440a73886f008cf903fcf5764ba8106a9fd5243d6f32729}"
```

Reply:
Exactly :slightly_smiling_face: